girder/api/sftp.py

```diff
@@ -220,7 +228,7 @@ class SftpServer(socketserver.ThreadingTCPServer):
 
     allow_reuse_address = True
 
-    def __init__(self, address, hostKey):
+    def __init__(self, address, hostKey, *args, **kwargs):
```

Comment:
I think we should either not make this variadic and only add `sftpServerAdapter=None`, or else pass through all variadic args into `socketserver.TCPServer.__init__` and explicitly document that in the docstring.

Reply:
I added the sftpServerAdapter arg.